src/components/Editor/index.js

```diff
@@ -788,6 +795,23 @@ class Editor extends PureComponent {
     });
   }
 
+  renderInScopeLines() {
+    const { linesInScope } = this.props;
+    if (
+      !isEnabled("highlightScopeLines") ||
```

Comment:
i'm re-adding a feature flag because i'm not happy with the cases we're missing like blocks and comments. Seeing those issues makes me think we should wait and re-launch highlighting when it's further along. 

With that said, we still get the onMouseOver logic so fewer accidental popups